app/views/comments/_comment.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if comment && comment.user %>
+<% if comment && comment.cached_user.present? %>
```

Comment:
Because we were already guarding against the absence of a user for any reason, we have this safety clause if any component of this change fails on deployment.